src/po/components/common/generateTitles.js

```diff
@@ -0,0 +1,14 @@
+export class GenerateRandomTitles {
```

Comment:
move this file to utils or helpers

Reply:
and it seems that we could make methods static, or export already created instance
export const stringGenerator = new RandomTitlesGenerator()